pallets/price-feed/src/lib.rs

```diff
@@ -260,6 +265,45 @@ pub mod pallet {
 		}
 	}
 
+	#[cfg(feature = "runtime-benchmarks")]
+	impl<T: Config> PriceFeedBenchmarks<T::AccountId, T::AssetId> for Pallet<T> {
+		fn create_feed(
+			caller: <T as frame_system::Config>::AccountId,
+			asset_id: T::AssetId,
+		) -> DispatchResultWithPostInfo {
+			use frame_benchmarking::vec;
+
+			pallet_chainlink_feed::Pallet::<T>::set_feed_creator(
+				<frame_system::Origin<T>>::Signed(pallet_chainlink_feed::Pallet::<T>::pallet_admin()).into(),
+				caller.clone(),
+			)?;
+
+			pallet_chainlink_feed::Pallet::<T>::create_feed(
+				<frame_system::Origin<T>>::Signed(caller.clone()).into(),
+				100u32.into(),
+				Zero::zero(),
+				(1u8.into(), 100u8.into()),
+				1u8.into(),
+				8u8,
+				vec![1; T::StringLimit::get() as usize],
+				Zero::zero(),
+				vec![(caller.clone(), caller.clone())],
+				None,
+				None,
+			)?;
+
+			let feed_id = <pallet_chainlink_feed::FeedCounter<T>>::get() - 1.into();
+			AssetFeeds::<T>::insert(&asset_id, feed_id);
+			pallet_chainlink_feed::Pallet::<T>::submit(
+				<frame_system::Origin<T>>::Signed(caller.clone()).into(),
+				feed_id,
+				1_u32.into(),
+				42.into(),
+			)?;
+			Ok(().into())
+		}
+	}
+
```

Comment:
This is essentially a mock?
I'm rather against adding this. Can you extend on why this is necessary, we should fix this upstream if this is a problem of the chainlink pallet.

Reply:
Yep, I'm thinking of fixing it in the chainlink repo and open another issue for this